i18n/en-US.properties

```diff
@@ -166,6 +168,8 @@ be.contentSidebar.activityFeed.taskNew.tasksFeedHeadlineApprovalCurrentUser = {u
 be.contentSidebar.activityFeed.taskNew.tasksFeedHeadlineGeneral = {user} assigned a Task
 # Comment headline for a general task assigned to the current user
 be.contentSidebar.activityFeed.taskNew.tasksFeedHeadlineGeneralCurrentUser = {user} assigned you a Task
+# Headline for a task assigned to a current user
```

Comment:
I don't think we need a string for this situation, but not sure.

@ehoogerbeets any thoughts on this scenario?

Reply:
Removing this as I can render the user directly.